.travis.yml

```diff
@@ -1,10 +1,7 @@
 # ref: https://docs.travis-ci.com/user/languages/python
 language: python
 python:
-  - "2.7"
-  - "3.3"
-  - "3.4"
-  - "3.5"
```

Comment:
Why are these versions removed?

Reply:
travis was installing python 3.3
and minimum version test was failing even before testing any python code

Reply:
just remove `3.3`, keep other versions.